scripts/stablex/setup_environment_for_single_trade.js

```diff
@@ -0,0 +1,37 @@
+// Prerequisit for this script:
+// Have an account funded with Rinkeby OWL 
+// This simple script will set up a trade between OWL and A New Token
+const StablecoinConverter = artifacts.require("StablecoinConverter")
+const ERC20Mintable = artifacts.require("ERC20Mintable.sol")
+const { addTokens, depositTokens, placeOrder } = require("./utilities.js")
+
+module.exports = async function (callback) {
+  try {
+
+    const instance = await StablecoinConverter.deployed()
+    const accounts = await web3.eth.getAccounts()
+    const account = accounts[0]
+
+    const TokenOWL = artifacts.require("TokenOWL")
+    const owlToken = await TokenOWL.at(await instance.feeToken.call())
+
+    const amount = (10 ** 18).toString()
+
+
+    const token = await ERC20Mintable.new()
+    await token.mint(account, amount)
+    await addTokens([token.address], web3, artifacts)
```

Comment:
Everything until here should not be needed, since we will do it as part of the migrations.

I think I'd prefer it if we paste the command line arguments we need to run the deposit/place order scripts into a readme. This way people can copy them for a simple/dumb trade but also easily adjust them if the want to trade different amounts/tokens.